xesmf/__init__.py

```diff
@@ -1,10 +1,7 @@
-# flake8: noqa
-from pkg_resources import DistributionNotFound, get_distribution
-
 from . import data, util
 from .frontend import Regridder, SpatialAverager
 
 try:
-    __version__ = get_distribution(__name__).version
-except DistributionNotFound:  # pragma: no cover
-    __version__ = '0.0.0'  # pragma: no cover
+    from ._version import __version__
+except ImportError:
+    __version__ = "unknown"
```

Comment:
0.0.0 gives the idea of a release. "unknown" is probably a better tag for something that is unreleased.